server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -105,6 +105,16 @@ it is around the label */
   @include u-height('105');
 }
 
+// Between sections
+.section-external {
+  @include u-margin-top(4);
+}
+
+// Between elements within a section
+.section-internal {
```

Comment:
question(non-blocker): is u-margin-top(3) going to be our default section margin within a section?

Reply:
Yes, see [the mocks](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=4093-7540&m=dev)